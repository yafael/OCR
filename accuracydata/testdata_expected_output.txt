couriernew_all.png,"123 abc ABC"
couriernew_helloworld_mixed.png,"Hello World"
couriernew_helloworld_upper.png,"HELLO WORLD"
couriernew_helloworld2_upper.png,"HELLO WORLD"
foobar.png,"FOO BAR"
licenseplate_upper+digits.png,"AF83HSK27"
multiline.png,"HELLO WORLD FOO BAR"
multiline_number.png,"H5LLO W8RLD F9O B4R"
timesnewroman_digits.png,"614 292 2121"
tnr_helloworld_lower.png,"hello world"
