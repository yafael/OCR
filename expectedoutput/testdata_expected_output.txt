couriernew_all.png,"123 abc ABC"
couriernew_helloworld_upper.png,"HELLO WORLD"
foobar.png,"FOO BAR"
licenseplate_upper+digits.png,"AF83HSK27"
multiline.png,"HELLO WORLD FOO BAR"
multiline_number.png,"H5LLO W8RLD F9O B4R"
timesnewroman_digits.png,"614 292 2121"
